src/markdown-pages/add-ons/rook.md

```diff
@@ -36,11 +30,17 @@ flags-table
 
 ## Block Storage
 
-For production clusters, Rook should be configured to use block devices rather than the filesystem.
-Enabling block storage is required with version 1.4.3+. Therefore, the `isBlockStorageEnabled` option will always be set to true when using version 1.4.3+.
-The following spec enables block storage for the Rook add-on and automatically uses disks matching the regex `/sd[b-z]/`.
-Rook will start an OSD for each discovered disk, which could result in multiple OSDs running on a single node.
-Rook will ignore block devices that already have a filesystem on them.
+For Rook versions 1.4.3 and later, block storage is required.
+For Rook versions earlier than 1.4.3, block storage is recommended in production clusters.
+
+You can enable and disable block storage for Rook with the `isBlockStorageEnabled` field in the kURL spec.
+By default, the `isBlockStorageEnabled` field is set to `true` for Rook versions 1.4.3 and later.
```

Comment:
that second section seems implicit from above. perhaps there is a better way to word it?

```suggestion
You can enable and disable block storage for Rook versions earlier than 1.4.3 with the `isBlockStorageEnabled` field in the kURL spec.
```

Reply:
@emosbaugh good catch. I think my question here was that it wasn't clear to me if the `isBlockStorageEnabled` field only exists in versions earlier than 1.4.3 (if so, I can update the flags table to make this more explicit)

Reply:
yes isBlockStorageEnabled only exists for earlier versions